### DISCUSSION TITLE
How to skip Getting Started screen and select a default template?

### DISCUSSION THREAD

**Discussion by santiagobasulto** [gitpod-io/openvscode-server#255] [Q&A] [answered]
I'm doing some tests with some students and I'd like to show them a predefined theme + skip the "Getting Started" screen altogether. What's the best way to do both those things:
* pre-select a template for a new user
* skip the Get Started screen?

![image](https://user-images.githubusercontent.com/872296/145079016-bacddac5-bb3e-4afa-b440-5e16f0168873.png)

Thanks!

**Comment by jeanp413** [comment, marked as answer]
@santiagobasulto by template you mean theme? For that you'll need to modify the source code and compile it yourself [here](https://github.com/gitpod-io/openvscode-server/blob/77d7e16e1973eff366e4b366db663b76e4dd26fa/src/vs/code/browser/workbench/workbench.ts#L488-L516) and update the `configurationDefaults` attribute with your desired settings

**Reply by santiagobasulto** [reply]
Ah, interesting. I thought it was something configurable via commandline attr or env var. I'll investigate further. Thanks @jeanp413.

**Reply by sullemanhossam** [reply]
Hi there i dont think i follow on how one would add a custom profile it seems to me the snippet does not install or prepare a profile for use

**Reply by sullemanhossam** [reply]
Its most likley something along the lines of 

[here](https://github.com/gitpod-io/openvscode-server/blob/7d16c5c7060d86e3b4c86c5226fa5aa00e077aad/src/vs/platform/userDataProfile/common/userDataProfile.ts#L4)
